### PR TITLE
Update VERSION for v0.7.0-dev

### PR DESCRIPTION
**Pull request by elton-furtado**

### CODE REVIEW

File: CHANGELOG/CHANGELOG-0.6.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Changelog 0.6
+
+## 0.5.0 - 2023-05-24
```
Comment (tushartathgur) [inline CHANGELOG/CHANGELOG-0.6.md:3]: nit: 
## 0.6.0 - 2023-05-24

File: VERSION_MAP [hunk 2]
```diff
@@ -4,3 +4,4 @@ v0.3.0 v1.9.0
 v0.4.0 v1.10.0
 v0.5.0 v1.11.0
 v0.6.0 v1.12.0
+v0.7.0 v1.13.0
```
Comment (tushartathgur) [inline VERSION_MAP:7]: Add Blank line

**Review by yanjunz97 — APPROVED** [review]
LGTM

**Review by tushartathgur — APPROVED** [review]
LGTM

**Pull request merged**